test/index.ts

```diff
@@ -73,13 +73,37 @@ describe('Mutation createUser', () => {
     const response = await request('localhost:4000').post('/').send(queryCreateUser);
     const message = response.body.errors[0].message;
 
-    expect(message).to.be.equal('Email is already in use');
+    expect(message).to.be.equal('Email is already in use.');
 
     const userRepository = User.getRepository();
     const users = await userRepository.find({ email: userPredefinedData.email });
 
     expect(users.length).to.be.equal(1);
   });
+
+  it('should not let password less than 6 characters long', async () => {
+    queryCreateUser.variables.data.password = 'abc';
```

Comment:
costumamos recomendar um padrao de testes AAA -> Arrange (prepara o cenario do teste) -> Act (executa o metodo/chamada a ser testado) -> Assert (compara os resultados). Que acaba funcionando como um feedback do proprio teste (se o arrange é mto grande, talvez de pra isolar coisa no teste, idealmente nao precisaria ter outro act dps do assert, etc.
ref: https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/
https://medium.com/@pablodarde/o-padr%C3%A3o-triple-a-arrange-act-assert-741e2a94cf88

esses novos testes estao dentro desse criterio ja, e ai acho que seria legal ter um espaçamento vertical pra facilitar a leitura e ficar claro (uma quebra de linha depois do arrange, outra dps do act, etc). tl;dr: uma quebra de linha dps da linha 85 e dps da linha 87 (mesma coisa para os outros its)

Reply:
concordo, costumo dar esses espaços mas por algum motivo aqui não fiz isso, já dei push nas mudanças